app/src/main/java/space/taran/arknavigator/mvp/model/UserPreferences.kt

```diff
@@ -16,6 +17,14 @@ class UserPreferences @Inject constructor(val context: Context) {
 
     private val dataStore = context.preferencesDatastore
 
+    suspend fun clearPreferences() {
+        setSorting(Sorting.DEFAULT)
+        setSortingAscending(true)
+        setCrashReportEnabled(null)
+        setCacheReplicationEnabled(true)
```

Comment:
Let's set unimplemented features to `false`.
Can we also make these toggles inactive (gray and non-switchable)?
Also a toast notification by pressing on them would be nice.

Reply:
> Let's set unimplemented features to `false`. Can we also make these toggles inactive (gray and non-switchable)? Also a toast notification by pressing on them would be nice.

As I understand, `setCrashReportEnabled(null)` should still remain the same, so that the app can set this feature by default based on the flavor of the build?

Reply:
@CloudLevi yes, that's fine here (we will have crash reporting soon).